bin/git-effort

```diff
@@ -80,41 +83,63 @@ color_for() {
 
 effort() {
     path=$1
+
+    # Exit if path == ""
+    # This is not an error
+    # shellcheck disable=SC2116
+    if [ -z "$(head -c 1 <<<"$(echo "$path")")" ]
+    then
+      exit 0
+    fi
```

Comment:
It looks like printf (that produces output to `effort`) sends a final empty line when we call `effort`. That's the reason for this check.